src/ui/ResponsiveComponents/ResponsiveDropdown/ResponsiveDropdownContent.ts

```diff
@@ -88,16 +90,41 @@ export class ResponsiveDropdownContent implements IResponsiveDropdownContent {
     this.element.setAttribute('aria-label', null);
   }
 
+  private get popperReferenceElement() {
+    return this.coveoRoot.find(`.${ResponsiveComponentsManager.DROPDOWN_HEADER_WRAPPER_CSS_CLASS}`);
+  }
+
   private createPopper() {
-    const referenceElement = this.coveoRoot.find(`.${ResponsiveComponentsManager.DROPDOWN_HEADER_WRAPPER_CSS_CLASS}`);
-    this.popperReference = new PopperJs(referenceElement, this.element.el, {
+    this.popperReference = new PopperJs(this.popperReferenceElement, this.element.el, {
       placement: 'bottom-end',
       positionFixed: true,
       modifiers: {
         preventOverflow: {
           boundariesElement: this.coveoRoot.el
+        },
+        computeStyle: {
+          gpuAcceleration: false
         }
-      }
+      },
+      eventsEnabled: false
+    });
+
+    this.bindPopperEvents();
+  }
+
+  private get scrollableParent() {
+    return ComponentOptions.findParentScrolling(this.popperReferenceElement);
+  }
+
+  private bindPopperEvents() {
+    $$(this.scrollableParent).on(['scroll', 'resize'], () => {
```

Comment:
Adding our own listeners doesn't create the issue in IE11

Reply:
Interesting, why is it that our custom listeners avoid the issue?

Reply:
Not exactly sure... I think it's because of how Popper resolves the scroll parents, it somehow doesn't resolve the dropdown and gets the window instead when scrolling through the dropdown in IE11. Actually it might even add event listeners on multiple scroll parents.

https://github.com/popperjs/popper-core/blob/v1.14.5/packages/popper/src/utils/setupEventListeners.js
https://github.com/popperjs/popper-core/blob/v1.14.5/packages/popper/src/utils/getScrollParent.js